README.md

```diff
@@ -48,4 +49,142 @@ signClient.request(payload).then((result) => {
 })
 ```
 
+By following these steps, developers can ensure a comprehensive and user-friendly integration of Hedera with WalletConnect in their dApps. Always refer to the linked documentation for in-depth details and best practices.
+
 ## Wallet Usage
+
+### 1. Setup and Installation
+
+First, make sure you've installed the necessary npm packages:
+
+```bash
+npm install @walletconnect/sign-client @hashgraph/sdk @hgraph.io/hedera-walletconnect-utils
+```
+
+If you have WalletConnect v1.x installed, consider following the [migration guide](https://docs.walletconnect.com/2.0/api/auth/wallet-usage) for a seamless transition to v2.x.
+
```

Comment:
this is for hedera specifically, and no wallet shoudl have wallet connect v1 integrated, so I think it makes sense to leave this out